### PR TITLE
[RN-48] "Help" menu option should open help link in a mobile browser

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
"Help" menu option should open help link in a mobile browser

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/RN-48
GitHub: https://github.com/mattermost/mattermost-mobile/issues/283

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization file updates

#### Device Information
This PR was tested on: Emulator Android 6.0

### CODE REVIEW

File: app/scenes/settings/settings.js [hunk 1]
```diff
@@ -129,6 +129,17 @@ export default class Settings extends PureComponent {
                             theme={theme}
                         />
                     }
+                    {config.HelpLink !== '' &&
```
Comment (enahum) [inline app/scenes/settings/settings.js]: you could do

```javascript
{config.HelpLink &&
... 
```

File: assets/base/i18n/en.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   "about.teamEditiont1": "Enterprise Edition",
   "about.title": "About Mattermost",
   "about.version": "Version:",
+  "help.title": "Help",
```
Comment (enahum) [inline assets/base/i18n/en.json]: does this key already exists? if so no need to add it here, if it does not exists it needs to have `mobile.` prefix like `mobile.help.title`

**Comment by enahum** [comment]
@cpanato were you able to fix this? I'm asking cause im about to do some changes that may affect this

**Comment by cpanato** [comment]
@enahum done. I did not have time yesterday.

File: app/scenes/settings/settings.js [hunk 3]
```diff
@@ -129,6 +129,17 @@ export default class Settings extends PureComponent {
                             theme={theme}
                         />
                     }
+                    {config.HelpLink &&
+                        <SettingsItem
+                            defaultMessage='Help'
+                            i18nId='mobile.help.title'
+                            iconName='help'
+                            iconType='material'
+                            onPress={() => Linking.openURL(config.HelpLink)}
```
Comment (enahum) [inline app/scenes/settings/settings.js]: can we surround this in a `preventDoubleTap` function?

like this

```javascript
preventDoubleTap(Linking.openURL, this, config.HelpLink)
```

thanks @cpanato

**Comment by cpanato** [comment]
@jarredwitt I didnt add that because in the example it stays together. 
Adding now per your request

**Pull request merged**